core/include/config.h

```diff
@@ -0,0 +1,65 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright 2017 Intel
+ * All rights reserved.
+ *
+ * Imported from Zephyr OS by:
+ * Rouven Czerwinski <r.czerwinski@pengutronix.de>
```

Comment:
Your name is in the commit message already.

Reply:
I'll drop this.